templates/galaxy/config/job_conf.yml

```diff
@@ -465,9 +465,9 @@ galaxy_jobconf:
   tools:
     # TODO(hxr): implement validation that checks that every destination
     # used here is defined above.
-    - id: "toolshed.g2.bx.psu.edu/repos/bgruening/keras_train_and_eval/keras_train_and_eval"
+    - id: keras_train_and_eval
       resources: usegpu
-    - id: "toolshed.g2.bx.psu.edu/repos/chemteam/gmx_sim/gmx_sim"
+    - id: gmx_sim
```

Comment:
Is regex pattern (`.*`) not required?

Reply:
Regex applies to TPV. This is the Galaxy job conf file. Based on the [documentation I linked](https://docs.galaxyproject.org/en/master/admin/jobs.html#static-destination-mapping), I do not think regex applies to the job conf.

Nevertheless I just gave it a try now on the GAT machine to be 100% sure:

```yaml
  tools:
    - class: local # these special tools that aren't parameterized for remote execution - expression tools, upload, etc
      environment: local_env
    - id: testing
      environment: tpv_dispatcher
      resources: testing
    - id: "toolshed.g2.bx.psu.edu/repos/bgruening/keras_train_and_eval/keras_train_and_eval/.*"
      resources: testing
```
Did not work :(

Reply:
Sorry! Wrong filename check.

Reply:
The long ID contains the version as well.